src/components/PaymentLog.vue

```diff
@@ -687,6 +697,8 @@ export default {
       salesCategories: [],
       hideOrderSummary: true,
       test1: true,
```

Comment:
what is test1?

Reply:
It is what we are using to popup the order summary modal on the right